cli/tests/test_disk_import.rs

```diff
@@ -736,7 +736,7 @@ fn test_disk_import_required_parameters() {
     bad_path.push("does.not.exist");
 
     let output = format!(
-        "path {} does not exist",
+        "Invalid Request: path {} does not exist",
```

Comment:
Having to make this change makes me wonder if we should just extra the error message to avoid some abstraction leaking here